pages/product/[productId]/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo, useProductList } from "@lib/hooks";
+
+const SidePanel = () => {
+  const router = useRouter();
+  const productId = Number(router.query?.productId);
+  const { error, isLoading, list = [] } = useProductList();
+  const { isLoading: isInfoLoading, product } = useProductInfo(productId, list);
+  const {
+    description,
+    is_visible: isVisible,
+    name,
+    price,
+    type,
+  } = product ?? {};
+  const panelData = { description, isVisible, name, price, type };
```

Comment:
this seems odd that we're taking destructured elements only to place them back into an object - we're effectively reproducing `product` here...